tcf_website/static/reviews/review_form.js

```diff
@@ -0,0 +1,132 @@
+        clearDropdown("#courseID");
+        clearDropdown("#instructor");
+        clearDropdown("#semester");
+        // Enable course selector, disable the following
+        $("#courseID").prop("disabled", false);
+        $("#instructor").prop("disabled", true);
+        $("#semester").prop("disabled", true);
+
+        // Fetch course data from API, based on selected subdepartment
+        var subdeptID = $("#subject").val();
+        var pageSize = "1000";
+        var courseEndpoint = "http://localhost:8000/api/courses/?subdepartment="
+                             + subdeptID + "&page_size=" + pageSize + "&recent";
+        $.getJSON(courseEndpoint, function(data) {
+            // Generate option tags
+            $.each(data.results, function(i, course) {
+                // 4-digit courseIDs only
+                if (course.number > 1000) {
+                    $("<option />", {
+                        val: course.id,
+                        text: course.number
+                    }).appendTo("#courseID");
+                }
+            });
+            return this;
+        });
+    });
+
+    // Fetch instructor data on course select
+    $("#courseID").change(function() {
+        // Clear & disable sequenced dropdowns
+        clearDropdown("#instructor");
+        clearDropdown("#semester");
+        // Enable instructor selector, disable the following
+        $("#instructor").prop("disabled", false);
+        $("#semester").prop("disabled", true);
+
+        // Fetch instructor data from API, based on selected course
+        var courseID = $("#courseID").val();
+        var pageSize = "1000";
+        var instrEndpoint = "http://localhost:8000/api/instructors/?section__course="
+                            + courseID + "&page_size=" + pageSize + "&recent=true";
+        $.getJSON(instrEndpoint, function(data) {
+            clearDropdown("#instructor"); // Empty dropdown
+
+            // Generate option tags
+            $.each(data.results, function(i, instr) {
+                $("<option />", {
+                    val: instr.id,
+                    text: instr.first_name + " " + instr.last_name
+                }).appendTo("#instructor");
+            });
+            return this;
+        });
+    });
+
+    // Fetch semester data on instructor select
+    $("#instructor").change(function() {
+        // Clear & disable sequenced dropdowns
+        clearDropdown("#semester");
+        // Enable semester selector, disable the following
+        $("#semester").prop("disabled", false);
+
+        // Fetch all semester data from API
+        var courseID = $("#courseID").val();
+        var instrID = $("#instructor").val();
+        var semEndpoint = "http://localhost:8000/api/semesters/?course="
+                          + courseID + "&instructor=" + instrID;
+        $.getJSON(semEndpoint, function(data) {
+            // Reverse chronological order (API default is chronological)
```

Comment:
Do want me to change this in the API?

Reply:
hmm maybe? it's not a huge issue though

Reply:
I just made the change myself (1 line change in api views)